src/misc/env_utils.cpp

```diff
@@ -0,0 +1,54 @@
+// SPDX-FileCopyrightText:  2022-2025 The DOSBox Staging Team
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
You put the header file in `src/utils` but the source file is `src/misc`. Both should be in `src/utils`.

Reply:
Sorry! I moved `env_utils`. I left `fs_utils` alone, best to clean that up in another PR in my mind

Reply:
That's fine, I'll do it later 😄